resources/assets/less/bem/btn-osu-big.less

```diff
@@ -51,8 +51,8 @@
   }
 
   &--account-edit {
-    display: block;
-    width: @avatar-size--base;
+    display: inline-block;
+    max-width: @avatar-size--base;
```

Comment:
did you mean `min-width`

Reply:
I did... my nonexistent dyslexia strikes again